src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -408,6 +412,21 @@ protected override void StopProcessing()
 
         #region Helper Methods
 
+        private void ThrowNotSuccessStatusCodeError(HttpRequestMessage request, HttpResponseMessage response)
+        {
+            using(var reader = new StreamReader(StreamHelper.GetResponseStream(response)))
```

Comment:
Thanks, I think it is Ok in this case.  My original concern is hiding the WebRequest exception with an IOException caused while processing the WebRequest exception.